app/screens/Onboarding/Parents/WelcomeParent/WelcomeParent.tsx

```diff
@@ -22,6 +23,7 @@ export const WelcomeParentScreen: React.FC<IWelcomeParentScreenProps> = ({
 
   const onLoginPress = useCallback(() => {
     navigation.navigate('Login');
+    trackButtonPress('Login');
   }, [navigation]);
```

Comment:
We should patch our ExtendedButton with analytic for ALL buttons, do not do it manually.

Reply:
in next pr